applications/FluidDynamicsApplication/python_scripts/compressible_slip_wall_process.py

```diff
@@ -70,6 +70,7 @@ def __init__(self, model, settings):
         self.rampup_period = settings["rampup_period"].GetDouble()
         self.recompute_normals = settings["recompute_normals"].GetBool()
         self._stage = self.AWAITING
+        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable"].GetString())
```

Comment:
```suggestion
        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable_name"].GetString())
```
To match the naming we normally use in Kratos.

Reply:
You should also check that the provided variable is of vector type.

Reply:
This is done during the `Check`.

Reply:
OK!